2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Configure stats pid to 4037
2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Loading settings from /Users/frederikmohr/.config/wandb/settings
2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Loading settings from /Users/frederikmohr/masterarbeit/NN_Forecasting/pytorch_lightning_files/wandb/settings
2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-14 13:19:18,651 INFO    MainThread:4037 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-14 13:19:18,652 INFO    MainThread:4037 [wandb_init.py:_log_setup():521] Logging user logs to ./wandb/run-20240414_131918-oka25w9f/logs/debug.log
2024-04-14 13:19:18,652 INFO    MainThread:4037 [wandb_init.py:_log_setup():522] Logging internal logs to ./wandb/run-20240414_131918-oka25w9f/logs/debug-internal.log
2024-04-14 13:19:18,652 INFO    MainThread:4037 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x31c4046a0>
2024-04-14 13:19:18,652 INFO    MainThread:4037 [wandb_init.py:init():561] calling init triggers
2024-04-14 13:19:18,652 INFO    MainThread:4037 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'dataset': 'The Coca-Cola Company closing prices', 'architecture': 'RNN', 'num_units': 40, 'num_layers': 1, 'dropout': 0, 'seq_length': 13, 'start_date': '2009-01-01', 'end_date': '2024-01-01', 'epochs': 30, 'learning_rate': 0.001}
2024-04-14 13:19:18,652 INFO    MainThread:4037 [wandb_init.py:init():611] starting backend
2024-04-14 13:19:18,652 INFO    MainThread:4037 [wandb_init.py:init():615] setting up manager
2024-04-14 13:19:18,654 INFO    MainThread:4037 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-04-14 13:19:18,655 INFO    MainThread:4037 [wandb_init.py:init():623] backend started and connected
2024-04-14 13:19:18,663 INFO    MainThread:4037 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-14 13:19:18,663 INFO    MainThread:4037 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-14 13:19:18,664 INFO    MainThread:4037 [wandb_init.py:init():715] updated telemetry
2024-04-14 13:19:18,694 INFO    MainThread:4037 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-14 13:19:19,235 INFO    MainThread:4037 [wandb_run.py:_on_init():2357] communicating current version
2024-04-14 13:19:19,329 INFO    MainThread:4037 [wandb_run.py:_on_init():2366] got version response 
2024-04-14 13:19:19,329 INFO    MainThread:4037 [wandb_init.py:init():799] starting run threads in backend
2024-04-14 13:19:20,276 INFO    MainThread:4037 [wandb_run.py:_console_start():2335] atexit reg
2024-04-14 13:19:20,277 INFO    MainThread:4037 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-14 13:19:20,277 INFO    MainThread:4037 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-14 13:19:20,277 INFO    MainThread:4037 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-14 13:19:20,277 INFO    MainThread:4037 [wandb_init.py:init():842] run started, returning control to user process
2024-04-14 13:19:41,309 INFO    MainThread:4037 [wandb_run.py:_finish():2064] finishing run frederik135/RNN_single_step_forecasts/oka25w9f
2024-04-14 13:19:41,310 INFO    MainThread:4037 [jupyter.py:save_history():445] not saving jupyter history
2024-04-14 13:19:41,310 INFO    MainThread:4037 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-14 13:19:41,310 INFO    MainThread:4037 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-14 13:19:41,310 INFO    MainThread:4037 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-14 13:19:41,310 INFO    MainThread:4037 [wandb_run.py:_restore():2287] restore
2024-04-14 13:19:41,310 INFO    MainThread:4037 [wandb_run.py:_restore():2293] restore done
2024-04-14 13:20:22,818 INFO    MainThread:4037 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-14 13:20:22,820 INFO    MainThread:4037 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-14 13:20:22,830 INFO    MainThread:4037 [wandb_run.py:_footer_sync_info():3895] logging synced files
